src/sentry/interfaces/stacktrace.py

```diff
@@ -36,6 +36,10 @@
 )/""", re.X | re.I
 )
 
+# Native function trim re.  For now this is a simple hack until we have the
+# language hints in which will let us trim this down better.
+_native_function_trim_re = re.compile(r'^(.[^(]*)\(')
```

Comment:
We shouldn't demangle argument lists and return types, to begin with, but the C++ demangle doesn't allow to customise this yet. Putting this on the TSC list.

Reply:
Not a huge fan of this hack. Definitely want to remember fixing this cleanly. I think we might benefit from making space in the frame for more extra metadata on the function.